config/config.go

```diff
@@ -183,6 +186,59 @@ func (c *Config) SetLogLevel(logLevel string) {
 	c.logLevel = logLevel
 	logger.SetLogLevel(c.logLevel)
 }
+func (c *Config) SetLogFilePath(logFilePath string) {
+	c.logFilePath = logFilePath
+}
+
+func (c *Config) SetLogOutput() error {
+	file, err := getFile(c.logFilePath)
+	if err != nil {
+		return err
+	}
+	logger.LogSetOutput(file, os.Stdout)
```

Comment:
I think that `os.Stdout`value should be a parameter, so we can test without override the global `os.Stdout` value.